### PR TITLE
Feature/d 10

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/HRIN-758

- Make module work in d-10

Awaiting:
https://github.com/itk-dev/pretix-api-client-php/pull/4

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -110,7 +110,7 @@ jobs:
         run: |
           # We need a Drupal project to run drupal-check (cf. https://github.com/mglaman/drupal-check#usage)
           # Install Drupal
-          composer --no-interaction create-project drupal/recommended-project:^9 --stability=dev drupal
+          composer --no-interaction create-project drupal/recommended-project:^10 --stability=dev drupal
```
Comment (rimi-itk) [inline .github/workflows/pr.yaml]: I've found a cleaner way to do this and we should consider updating this all over the place; see https://github.com/os2forms/os2forms#code-analysis (and https://github.com/OS2Forms/os2forms/blob/develop/scripts/code-analysis) for details.

File: itk_pretix.info.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: 'ITK Pretix'
 type: module
 description: 'A module using pretix API to generate events from field values'
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^8 || ^9 || ^10
```
Comment (rimi-itk) [inline itk_pretix.info.yml]:
```suggestion
core_version_requirement: ^10
```

**Review by rimi-itk — APPROVED** [review]
Approved with some comments.

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (martinyde) [reply .github/workflows/pr.yaml]: Changed to use script.

File: itk_pretix.info.yml (hunk 2 above)
Comment (martinyde) [reply itk_pretix.info.yml]: Changed to run on d9 or d10
This way the module can be updated seperately from drupal

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**